app/models/pod.rb

```diff
@@ -41,6 +41,10 @@ class Pod < ApplicationRecord
 
   validate :not_own_pod
 
+  scope :possible_available , -> {
+    where(["offline_since is null or offline_since > ?", Time.now - 14.days])
```

Comment:
Rails/TimeZone: Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.xmlschema`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.httpdate`, `Time.now.to_i`, `Time.now.to_f` instead.

Reply:
I probably would just have used `DateTime.now.utc - 14.days` as it's already used here (but that's irrelevant now anyway, as I wouldn't filter them in the backend, so this isn't needed):

https://github.com/diaspora/diaspora/blob/d9c72edb520a12d70f9c17329a743f5ea6b81e78/app/models/pod.rb#L81-L83

By the way, the naming we used for this until now was "active" (so pods who are down for less than 14 days still count as active), and you can probably just use this method that already exists and add it to the `PodPresenter` so the frontend can access the flag, so you don't need to compare timestamps in the frontend and it uses the exact same logic as in the backend. That way if we ever change the numbers of days, we still only have this one place and don't need to change it at dfifferent places.